test/Sentry.Log4Net.Tests/SentryAppenderTests.cs

```diff
@@ -291,6 +309,32 @@ public void Append_ConfiguredEnvironment()
             .CaptureEvent(Arg.Is<SentryEvent>(e => e.Environment == expected));
     }
 
+    [Fact]
+    public void MinimumEventLevel_DefaultsToNull()
+    {
+        var appender = new SentryAppender();
+        Assert.Null(appender.MinimumEventLevel);
+    }
+
+    [Fact]
+    public void MinimumEventLevel_ConvertsEventsBelowToBreadcrumbs()
```

Comment:
For [naming unit tests](https://github.com/getsentry/sentry-dotnet/blob/main/CONTRIBUTING.md#naming-tests), we try to use the convention `Method_Context_Expectation`. For this one then, maybe `DoAppend_BelowMinimumEventLevel_AddsBreadcrumb`.

Reply:
Renamed ✅